test/jest/setup.js

```diff
@@ -8,3 +9,8 @@ jest.mock('webextension-polyfill', () => {
     },
   };
 });
+
+/* eslint-disable-next-line jest/require-top-level-describe */
+beforeEach(() => {
```

Comment:
We may be able to apply this to our remaining Mocha unit tests, but is there an equivalent setup file for those tests?

Reply:
Yes, there is a setup file that is used for all unit tests (including Jest and Mocha). Mutiple setup files actually, for reasons that I don't understand.

I was going to suggest adding this to `test/helpers/setup-helper.js` because we already interact with Nock there.

Reply:
Oh wait, I see that this is invoked by "setupFilesAfterEnv". Maybe it needs to be here

Reply:
Ah, good call. Done in [aebf425](https://github.com/MetaMask/metamask-extension/pull/15717/commits/aebf4254be004af92da410c60ff29acf8c0eb030).